### DISCUSSION TITLE
Type error for Hst prop in Svelte

### DISCUSSION THREAD

**Discussion by lachlancollins** [histoire-dev/histoire#272] [Q&A] [answered]
Hi, I've just tried out Histoire in my SvelteKit project, and I'm trying to fix the TS (actually type-checked JS) warning on the Hst prop. It's currently causing the `svelte-check` command to fail. I tried searching through this repo for a type but couldn't find anything. It also affects the `examples/sveltekit` project.

`Variable 'Hst' implicitly has type 'any' in some locations where its type cannot be determined.js(7034)`.

![image](https://user-images.githubusercontent.com/1667261/186919842-8d9439a1-e2c2-43a2-95b7-cbd17d9ec841.png)

**Comment by Akryum** [comment]
Your can try putting `:any` for now

Le ven. 26 août 2022, 15:56, Lachlan Collins ***@***.***> a
écrit :

**Comment by vnphanquang** [comment, marked as answer]
This now works for me:

```html
<script lang="ts">
  import type { Hst } from '@histoire/plugin-svelte';

  export let Hst: Hst;
</script>
```

**Reply by corydeppen** [reply]
Awesome! I just noticed this was also mentioned in #289 ([comment](https://github.com/histoire-dev/histoire/discussions/289#discussioncomment-3783768)) and released in [`0.11.0`](https://github.com/histoire-dev/histoire/releases/tag/v0.11.0).
